pigpio.js

```diff
@@ -376,10 +376,12 @@ Gpio.prototype.waveDelete = function(waveId) {
 	pigpio.gpioWaveDelete(waveId);
 	return this;
 };
-
 Gpio.prototype.waveTxSend = function(waveId, waveMode) {
 	return pigpio.gpioWaveTxSend(waveId, waveMode);
 };
+Gpio.prototype.waveAddSerial = function(user_gpio, baud, data_bits, stop_bits, offset, numBytes, str) {
+	return pigpio.gpioWaveAddSerial(user_gpio, baud, data_bits, stop_bits, offset, numBytes, str);
```

Comment:
I know the original pigpio is inconsistent, but could you please stick to one style... preferably camel case.